CHANGES.txt

```diff
@@ -1545,22 +1553,4 @@ Features
 * Image descriptions in galleries
 * Image exclusion in galleries
 * Special "draft" tag
-* Pretty code listings ("code galleries")
```

Comment:
PyCharm remove part of the file (?) and I didn't see if before commiting/pushing.